File: csharp/fizzbuzz/FizzBuzzEngine/FizzRule.cs
using System;
namespace FizzBuzzEngine
{
    public class FizzRule : BaseRule, IRule
    {
        public FizzRule()
        {
        }

        public string Convert(int number)
        {
            return IsMultiple(number, 3) ? "fizz" : "";
        }
    }
}

File: java/fizzbuzz-openclose/src/main/java/org/okiwi/codingdojo/FizzRule.java
package org.okiwi.codingdojo;

public class FizzRule implements IRule {
    public FizzRule() {
    }

    @Override
    public String test(Integer number) {
        if (number % 3 == 0) {
            return "fizz";
        }
        return "";
    }
}